packages/dashboard/src/portal/components/VotePopup.vue

```diff
@@ -44,6 +55,13 @@ export default class VotePopup extends Vue {
     this.openProposalDialog = true;
   }
 
+  async getUserNodes(farmIDs: any[]): Promise<any> {
```

Comment:
we don't send the twin id to this method so I think the method name is invalid. maybe we can call it `getFarmsNodes`